src/test/java/com/catapi/service/TranslationServiceTest.java

```diff
@@ -0,0 +1,201 @@
+
+        ArgumentCaptor<CatFactTranslation> breedCaptor = ArgumentCaptor.forClass(CatFactTranslation.class);
+        verify(catFactTranslationRepository).save(breedCaptor.capture());
+
+        CatFactTranslation updatedTranslation = breedCaptor.getAllValues().getFirst();
+
+        CatFactTranslation expectedUpdated = new CatFactTranslation();
+        expectedUpdated.setLocale(Locale.RU);
+        expectedUpdated.setTranslationText("Обновленный текст");
+        expectedUpdated.setUpdateMode(UpdateMode.MANUAL);
+
+        assertEquals(expectedUpdated, updatedTranslation);
+    }
+
+
+    private void mockOkHttpClientResponse(String originalText, String expectedTranslation) {
+        ResponseBody responseBody = ResponseBody.create(
+                getTranslationResponse(expectedTranslation),
+                MediaType.get("text/javascript; charset=utf-8")
+        );
+
+        // Создаем объект ответа
+        Response response = new Response.Builder()
+                .request(new Request.Builder().url(LINGUATOOLS_URL).build())
+                .protocol(Protocol.HTTP_1_1)
+                .code(200)
+                .message("OK")
+                .body(responseBody)
+                .build();
+
+        Call call = mock(Call.class);
+        try {
+            when(call.execute()).thenReturn(response);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        when(okHttpClient.newCall(argThat(request -> {
+            String query = request.url().toString();
+            return URLDecoder.decode(query, StandardCharsets.UTF_8).contains(originalText);
+        }))).thenReturn(call);
+    }
+
+    private String getTranslationResponse(String testToTranslate) {
+        return String.format("""
+                \t$('#translate_input').css("visibility","");
+                \t$('#translate_input').css("visibility","visible");
+                \t//console.log("@trans: text");
+                \t\t$('#detected_language').empty();
+                \t\t$('#detected_language').removeAttr("code_source");
+                \t\t$('#detected_language').removeClass("alert-info -p-2");
+                \t$('#trans_output_content').html("");
+                \t$('#trans_output_content').html("%s");
+                \t$('#translated_by').empty();
+
+                \t$('#lt_trans_header').css("visibility","visible");
+                \t$('#lt_trans_l1').empty();
+
+                \t$('#startseite_deeplink').css("visibility","visible");
+                \t$('#deeplink').empty();
+
+
+                \tvar translated_by_name = "Google Translate";
+                \tvar translated_by_url = "https://translate.google.com/";
+                \t$('#translated_by').html("Translated by <a href=" + translated_by_url + " target='_blank' rel='nofollow' class='text-secondary' style='text-decoration:underline;'>" + translated_by_name + "</a>");
+                """, testToTranslate);
+    }
```

Comment:
This is how Linguatool endpoint repsonses =)

So there is no way except parse this sting